addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -17,9 +17,8 @@
                         <field name="has_multiple_acquirers" invisible="1"/>
                         <field name="description"/>
                         <field name="amount"/>
-                        <field name="acquirer_id"
-                            placeholder="Leave empty to allow all acquirers"
-                            options="{'no_open': True, 'no_create': True}"
+                        <field name="payment_acquirer_selection"
+                            widget="selection"
```

Comment:
Isn't it already the default?

Reply:
Yes indeed, I removed it after testing that it didn't have any impact.